tests/auth/test_auth_store.py

```diff
@@ -1,10 +1,10 @@
 """Tests for the auth store."""
 import asyncio
 
-import asynctest
-
 from homeassistant.auth import auth_store
 
+import tests.async_mock
```

Comment:
I think we should do `from tests.async_mock import patch` so that the diff will be minimal in the future when we'll remove the helper.